### ISSUE TITLE
Overlay transparency slider can be switched off by underlay and vice versa

### ISSUE THREAD

**Issue by mikehgentry** [osmandapp/OsmAnd#16643]
### Description

I mentioned this before in #10077, but I'm closing that because it's a bit unfocused. This bit is undoubtedly a bug: there's only one transparency slider whether you have an overlay, underlay or both. This can lead to a disparity between settings and its actual state.

### Steps to reproduce

Create an underlay, with the transparency slider on. Create an overlay with the slider off.

### Actual result

There is now no slider, even though the underlay setting is switched to on. Disabling the overlay doesn't make it reappear, but disabling and reenabling the underlay does.

### Expected result

If only one *lay has the setting switched to on, the slider should exist and be attached to that *lay. If both *lays have it set to on, it's less clear what should be done. I'd argue for just stacking two of them above one another, or having a switch to one side (or long press or something) to change between the two. Or if necessary just disallow having sliders for both enabled at the same time in the settings.

### Your Environment
OsmAnd Version: + 4.4.1 beta
Android/iOS version:  11
Device model:  Xcover 4s

**Comment by mikehgentry** [comment]
The presence / absence of the slider seems to also get messed up when switching profile sometimes. When this happens the UI for switching the layer off/on is out of sync (the button says it's on, but when you press it it thinks it's switching to on)

I'm not 100% how to reproduce yet - will update if I figure it out

**Comment by vshcherb** [comment]
Yes both needs to be displayed then as in IOS version

**Comment by letypequividelespoubelles** [comment]
It would be nice to be able to move the two sliders where we want in the screen, as with the quick action button. Thanks !

**Comment by Gregor0000** [comment]
Yes, I agree. Two sliders for separate control of overlay and underlay transparency would be great.
